src/components/MapboxMap/MapboxMap.tsx

```diff
@@ -9,14 +9,17 @@ import { MapStyle, mapStyles } from './mapStyles';
 
 export type MapboxMapProps = {
   children?: React.ReactNode;
-  ref?: MutableRefObject<MapRef | null>;
   // kind of ugly, but "MapProps" has a small bug which makes it unusable
 } & Partial<React.ComponentProps<typeof Map>>;
 
 /**
  * Renders a Mapbox GL JS Map with our access key and sensible defaults
  */
-export function MapboxMap({ children, ...rest }: MapboxMapProps) {
+//const MyInput = forwardRef(function MyInput(props, ref) {
```

Comment:
Suggest removing comment (unless its for documentation?)
```suggestion
```

Reply:
good catch, was a debug artifact